profilegen/src/res/templates/spec_template.template

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative 'lib/client'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'azure_mgmt_profile_<%= profile_version %>'
+  spec.version       = '<%= profile_ruby_version %>'
+  spec.authors       = 'Microsoft Corporation'
+  spec.email         = 'azrubyteam@microsoft.com'
+  spec.description   = 'Microsoft Azure Management Profile'
+  spec.summary       = 'Official Ruby client library to consume Microsoft Azure Management profile.'
+  spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
+  spec.license       = 'MIT'
+
+  spec.files         = Dir['LICENSE.txt', 'lib/**/*']
```

Comment:
I double that this would work...we may need following in the top of this file:

```
lib = File.expand_path('../lib', __FILE__)
$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
```

Reply:
did you mean to say "I doubt"?, yeah, we need to try this out, is there a test checking on this?